src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,65 @@
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import org.junit.jupiter.api.Test;
+import dave.task.Deadline;
+import dave.exceptions.InvalidDateTimeFormatException;
+import dave.exceptions.InvalidDescriptionException;
+
+public class DeadlineTest {
+
+    @Test
+    public void testDeadlineCreation_validDateTime() throws InvalidDescriptionException, InvalidDateTimeFormatException {
+        Deadline deadline = new Deadline("Submit report /by 2024-08-02 1800");
+        LocalDate expectedDate = LocalDate.of(2024, 8, 2);
+        LocalTime expectedTime = LocalTime.of(18, 0);
+
+        assertEquals(expectedDate, deadline.dueDate);
+        assertEquals(expectedTime, deadline.dueTime);
+    }
+
+    @Test
+    public void testDeadlineCreation_invalidDate() {
```

Comment:
Test method should be renamed to follow this 3 part format: `featureUnderTest_testScenario_expectedBehavior()`
e.g. `sortList_emptyList_exceptionThrown()`, `getMember_memberNotFound_nullReturned()`